content/v2.0/reference/key-concepts.md

```diff
@@ -0,0 +1,184 @@
+|:-------------------  |:------------|:--|:---|:-------|:------|
+| 2019-08-18T00:00:00Z | census|bees |23 | 1  |anderson|
+| 2019-08-18T00:00:00Z | census|bees |30 | 1  |mullen  |
+| 2019-08-18T00:06:00Z | census|bees |28 |  2 |anderson|
+|                      |       |     |   |    |        |       |
+| <span class="tooltip" data-tooltip-text="Timestamp">2019-08-18T00:06:00Z</span>| <span class="tooltip" data-tooltip-text="measurement"> census</span>| <span class="tooltip" data-tooltip-text="Field key">ants</span>| <span class="tooltip" data-tooltip-text="Field value">3</span> | <span class="tooltip" data-tooltip-text="Tag value">2</span> |<span class="tooltip" data-tooltip-text="Tag value">mullen</span>|
+
+#### Timestamp
+
+All data stored in InfluxDB has a `_time` column that stores timestamps. Timestamps show the date and time in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) UTC associated with data. Timestamp precision is important. When you search data within a specified time interval, make sure the timestamp precision you're searching matches the timestamp precision in your dataset.
+
+#### Measurement
+
+The  `_measurement` column shows the name of the measurement `census`. Measurement names are strings. A measurement acts as a container for tags, fields, and timestamps. Use a measurement name that describes your data. The name `census` tells us that the field values record the number of `bees` and `ants`. A single measurement can belong to different [buckets](#bucket).
+
+#### Fields
+
+A field includes a field key stored in (`_field`) and associated field value(s) stored in (`_value`).
+
+##### Field key
+
+The field keys `bees` and `ants` is a string that stores the name of the field.
+
+##### Field values
+
+The field values are your data; they can be strings, floats, integers, or Booleans. A field value always has an associated timestamp. The field values in the sample data show the number of `bees` at specified times: `23`, `30`, and `28` and the number of `ants` at a specified time: `3`.
+
+##### Field sets
+
+A field set is a collection of field key-value pairs. The sample data includes the following four field sets:
+
+- `bees = 23`
+- `bees = 30`
+- `bees = 28`
+- `ants = 3`
+
+#### Fields aren't indexed
+
+Fields are required in InfluxDB data and are not indexed. Queries that filter field values must scan all field values to match query conditions. As a result, queries on tags are more performant than queries on fields. Store commonly queried metadata in tags.
+
+#### Tags
+
+The last two columns in the sample data, `location` and `scientist`, are tags.
+Tags include tag keys and tag values that are stored as strings and metadata.
+
+##### Tag keys
+
+The tag keys in the sample data are `location` and `scientist`.
+
+##### Tag values
+
+The tag key `location` has two tag values: `1` and `2`.
```

Comment:
two values: `loc1` and `loc2`.

Reply:
As above, consider proper names for locations.